config/environments/production.rb

```diff
@@ -80,6 +80,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.cache_store = :dalli_store, nil, {
+    username: ENV["MEMCACHIER_USERNAME"],
+    password: ENV["MEMCACHIER_PASSWORD"],
```

Comment:
We don't have a username or password in staging/production. We just have one ENV variable: `MEMCACHED_ENDPOINT` that contains the domain name and port.